tests/test_view.py

```diff
@@ -58,7 +62,90 @@ def test_view_zarr_with_napari(
     ),
     example_data_for_image_tests,
 )
-def test_view_ometiff_with_napari(
+def test_view_ometiff_with_napari_and_example_data(
```

Comment:
not sure if it's PEP8 policies to not include two new line characters after function definitions in decorated tests?

Reply:
Thanks - good point! I couldn't find a reference in [PEP8](https://peps.python.org/pep-0008/) about what to do when there are decorators at hand. `ruff-format` automatically formatted things in this case so I'll plan to stick with this (we'd have to provide an exemption / ignore for each place we do this if not).

Getting into the deep end: in my mind, decorators "wrap" a function, effectively creating a "new" definition of the function. That being said, if we think of the `@decorator` like a `def`, we'd still abide the two line rule between functions. Maybe this is how `ruff-format` interprets the rule as well.